src/FunctionApps/python/tests/FhirServerExport/test_main.py

```diff
@@ -13,13 +12,12 @@
 }
 
 
-@mock.patch("FhirServerExport.fhir.download_from_export_response")
 @mock.patch("FhirServerExport.fhir.export_from_fhir_server")
-@mock.patch.object(AzureFhirServerCredentialManager, "get_access_token")
+@mock.patch("FhirServerExport.AzureFhirServerCredentialManager")
```

Comment:
From @bamader : Another import flag (sorry if this is obnoxious, just trying to get ahead of changes I know will need to happen)

Response from @spencer-skylight : Addressed